docs/user-guide/release-notes/2.12.x.rst

```diff
@@ -25,6 +25,10 @@ New Features
   the result on the orphan removal task. This count is broken down by content unit type. In
   pulp-admin, the result can be looked up with the ``tasks details`` command.
 
+* Failover events for instances of pulp_resource_manager and pulp_celerybeat are now logged at the
+  WARNING level with a message which explicitly states that failover has occurred, along with the
```

Comment:
Can this be broken into two sentences?  I suggest s/level with a message which/level. The log message/